test/katex-spec.js

```diff
@@ -2843,4 +2843,10 @@ describe("Symbols", function() {
     it("should parse \\text{\\i\\j}", () => {
         expect("\\text{\\i\\j}").toParse();
     });
+
+    it("should render ligature commands like their unicode characters", () => {
+        const commands = getBuilt("\\text{\\ae\\AE\\oe\\OE\\o\\O\\ss}");
+        const unicode = getBuilt("\\text{æÆœŒøØß}");
+        expect(commands).toEqual(unicode);
```

Comment:
You can use `toParseLike` instead of this: `expect("\\text{...}").toParseLike("\\text{...}")`

Reply:
I tried that, but they don't actually parse the same.

Reply:
Ah, I see, `getBuilt` renders to HTML.  I didn't realize that -- it suggests some other tests we could do!